src/main/java/core/basesyntax/DataParserFormatter.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class DataParserFormatter {

```

Comment:
I think this is overkill for this task. You don't need to handle exception and have dedicated parser class, just use LocalDate.parse() and DateTimeFormatter.ofPattern in SalaryInfo.java